### ISSUE TITLE
Response Function Selection and Deletion

### ISSUE THREAD

**Issue by shantimorrell** [radiantlab/HDRICalibrationTool#28]
> As a user, I want to upload a camera response function, with the option to remove it if I upload the wrong file, so that I can generate an HDR image.

## Background

This is a subtask of #7.

This is the second step of generating an HDR image, coming immediately after uploading the LDR images. Each specific camera has its own camera response function which describes the relationship between the RGB values and the scene’s radiance values.

Therefore, the camera response function is a necessary part of the HDR image generation when using .jpg or .jpeg images as the input LDR images. 

Additional information about the HDR image generation process can be found here: [HDR Image Generation Tutorial](https://www.tandfonline.com/doi/full/10.1080/15502724.2019.1684319).

## Requirements

- The app will have one main page. In the second portion of the page, beneath the LDR image selection, there will be an input area for selecting the camera response function.
- The user can click the button to open a file dialog window that will allow them to browse files and select a camera response function file. 
- The camera response function will be stored as a React state. 
- The user can click a button to remove the uploaded camera response function if desired. 
- The file name (or perhaps full file path?) of the selected camera response function is displayed.

## Design Resources

No design resources exist yet that depict selecting a camera response function. However, the selection mechanism should be very similar to that of uploading LDR images, and a preliminary Figma design can be found here: [Figma Prototype](https://www.figma.com/proto/4z8auRhmcUPepPXcd3xxHM/Desktop-Interface?page-id=0%3A1&type=design&node-id=45-2050&viewport=-4070%2C427%2C0.51&t=qCTMZkk8mLYrhpIt-1&scaling=scale-down&starting-point-node-id=45%3A2050).

The prototype needs to be updated to depict displaying the file name (or file path) of the selected camera response function.

## Dependencies

The following need to be completed before this issue:

- [Initialize fork with Tauri/Next/TailwindCSS #18](https://github.com/radiantlab/HDRICalibrationTool/issues/18)

Before being able to run the hdrgen step of the pipeline, the following feature is also required:

- [Image selection, listing, previews, and deletion #19](https://github.com/radiantlab/HDRICalibrationTool/issues/19)

## Subtasks

None

## Estimate

1 week.

## Open Questions

Do we want to display only the file name or display the whole path of the selected camera response function?

What layout or grouping of camera settings and camera response function in our user interface makes the most sense?

## Acceptance Criteria

1. The user opens the app.
2. The user selects the Upload Camera Response Function button.
3. The dialog window opens and the user can select the file corresponding to the camera response function.
4. The user confirms the selected file in the dialog window.
5. The name of the file used for the camera response function is displayed.
6. The user can click the Remove button to remove the uploaded file.
7. The user can click the upload button again to find a different file.
8. The file selection dialog window opens again.
9. The user confirms the selected file in the dialog window.
10. The name of the new uploaded file is displayed under the camera response function section.

Stretch goal: Allow the camera response function to be dragged-and-dropped instead if desired.

## Definition of Done

Same as listed in the [Software Development Process](https://docs.google.com/document/u/1/d/1b9ew4qtVZ9rGZiTvdeFDgCCmq4kLCokuLqpRv2HTGIM/edit). No additional criteria.
